server/rpc/rpcadapters.go

```diff
@@ -239,8 +239,8 @@ var _ rpcserverSyncManager = (*rpcSyncMgr)(nil)
 //
 // This function is safe for concurrent access and is part of the
 // rpcserverSyncManager interface implementation.
-func (b *rpcSyncMgr) IsCurrent() bool {
-	return b.syncMgr.IsCurrent()
+func (b *rpcSyncMgr) IsSynced() bool {
```

Comment:
In case go vet doesn't catch this, please update the comment with the function's new name.